Pod/Classes/ValueStepper.swift

```diff
@@ -124,6 +118,7 @@ private enum Button: Int {
     
     public required init?(coder aDecoder: NSCoder) {
         super.init(coder: aDecoder)
+        translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Not sure about this line. It's usually supposed to be set by the user of the class, no?